src/ol/geom/geometry.js

```diff
@@ -132,6 +132,7 @@ ol.geom.Geometry.prototype.getClosestPoint = function(point, opt_closestPoint) {
 /**
  * @param {ol.Coordinate} coordinate Coordinate.
  * @return {boolean} Contains coordinate.
+ * @api
```

Comment:
Note: for `ol.geom.Polygon`, this algorithm is used: https://www.ecse.rpi.edu/Homepages/wrf/Research/Short_Notes/pnpoly.html